aleth-vm/main.cpp

```diff
@@ -113,6 +113,7 @@ int main(int argc, char** argv)
     addTransactionOption("input", po::value<string>(), "<d> Transaction code should be <d>");
     addTransactionOption("code", po::value<string>(),
         "<d> Contract code <d>. Makes transaction a call to this contract");
+    addTransactionOption("codeFile", po::value<string>(), "<d> File containing EVM code <d>. If '-' is specified, code is read from stdin");
```

Comment:
The `evm` tool from go-ethereum names it `--codefile`, so all-lowercase name might be preferred.

Reply:
Did the amendment on lowercase `--codefile`.